nukebuild/Build.cs

```diff
@@ -81,25 +73,23 @@ protected override void OnBuildInitialized()
                     .AddProperty("PackageVersion", Parameters.Version)
                     .AddProperty("iOSRoslynPathHackRequired", "true")
                     .SetToolsVersion(MSBuildToolsVersion._15_0)
-                    .AddTargets("Build")
```

Comment:
You've just made adding new lines here to cause possible merge-conflicts for no reason. People are even ignoring commas at the end of list to avoid them

Reply:
Reverted changes.